사용자, 라벨, 마일스톤 리스트 api 생성

rolled-potatoes opened the pull request:
- 더미로 사용할 데이터를 `server/src/service/dumy` 에 모아서 작성하였습니다. 
- GET '/user' : 사용자 리스트 
- GET '/label' : 라벨 리스트
- GET '/milestone' : 마일스톤 리스트 
---
- 현재 import 랑 require 를 같이 쓰는 부분이 있는데 추후에 수정해서 올릴 예정입니다.

rolled-potatoes committed:
Feat : user 라우트 생성

사용자 관련 리스트를 가져오기 위한 서버 라우팅 작업 함.

rolled-potatoes committed:
Feat : 사용자 리스트 api 작성

rolled-potatoes committed:
Merge remote-tracking branch 'upstream/master' into feature/userList

rolled-potatoes committed:
Fix : user service 오타 수정

rolled-potatoes committed:
Feat : label List api 작성

rolled-potatoes committed:
Feat : milestone 리스트 api 작성

rolled-potatoes committed:
Docs : 사용자, 라벨, 마일스톤 api 명세서 작성

changheedev reviewed (commented):
rest api는 `/users, /issues` 처럼 리소스를 복수형으로 사용하는 것을 권장하고 있는데요
fs로 파일을 불러오다보니 단수형태로 url이 생성되는것 같습니다. 
혹시 이렇게 맞추려면 파일명을 수정해야 할까요?

rolled-potatoes commented:
> rest api는 `/users, /issues` 처럼 리소스를 복수형으로 사용하는 것을 권장하고 있는데요
> fs로 파일을 불러오다보니 단수형태로 url이 생성되는것 같습니다.
> 혹시 이렇게 맞추려면 파일명을 수정해야 할까요?

파일명을 라우팅하는 방식을 하고 있는데 파일명을 바꾸던지 or 그냥 import 를 수동으로 해야 할 것 같아요

changheedev commented:
그러면 파일명만 바꾸는게 덜 번거로우니까 route쪽만 파일명을 복수형으로 쓰는걸로 해주세요

changheedev commented on server/src/service/issue.js:
```diff
@@ -2,8 +2,11 @@ const fs = require('fs');
 const path = require('path');
 
```
import 문으로 수정 부탁드립니다

rolled-potatoes committed:
Refactor : 라우팅 경로 변경 및 import/export 문법 변경

기존에 require를 사용하는 것을 import 로 통일 하기 위해 변경을 하였음.
route/index 에서 dynamic import eslint 에러로 인하여 bable-eslint 추가 함.
